modules/minimint-mint/src/lib.rs

```diff
@@ -313,6 +316,23 @@ impl FederationModule for Mint {
             .copied()
             .collect();
 
+        let mut redemptions = Amount::from_sat(0);
+        let mut issuances = Amount::from_sat(0);
+        self.db
+            .find_by_prefix(&MintAuditItemKeyPrefix)
+            .for_each(|res| {
+                let (key, amount) = res.expect("DB error");
+                match key {
+                    MintAuditItemKey::Issuance(_) => issuances += amount,
+                    MintAuditItemKey::IssuanceTotal => issuances += amount,
+                    MintAuditItemKey::Redemption(_) => redemptions += amount,
+                    MintAuditItemKey::RedemptionTotal => redemptions += amount,
```

Comment:
This is very clever (delete+insert instead of fetch in the beginning and then overwrite)!